### ISSUE TITLE
Pod won't compile

### ISSUE THREAD

**Issue by romaonthego** [mapbox/DEPRECATED-mapbox-ios-sdk#230]
I'm getting this error:

``` objective-c
undefined symbols for architecture i386:
  "_pj_free", referenced from:
      -[RMProjection dealloc] in libPods.a(RMProjection.o)
  "_pj_fwd", referenced from:
      -[RMProjection coordinateToProjectedPoint:] in libPods.a(RMProjection.o)
  "_pj_init_plus", referenced from:
      -[RMProjection initWithString:inBounds:] in libPods.a(RMProjection.o)
  "_pj_inv", referenced from:
      -[RMProjection projectedPointToCoordinate:] in libPods.a(RMProjection.o)
```

Also for some reason I need to add `CoreLocation` and `QuartzCore` manually into a project.

I'm using the latest CocoaPods 0.17.2, this a warning I get when install the `MapBox` pod:
`Specification#config is deprecated. The config is accessible from the parameter passed to the hooks`.

You can reproduce it by creating a new project.

**Comment by incanus** [comment]
Hmm, I hadn't heard of problems with this yet. I just updated the spec this week for 0.17.0+ and tested up to 0.17.1. Are you using the podspec in the repo or installing from the specs repo via the normal method? 

It looks to me like a linking issue with the `libProj4.a` that's included statically in our source. Will investigate.

**Comment by romaonthego** [comment]
Normally I would write `pod 'MapBox', '~> 1.0.2'` in my Podfile, but I tried this as well:
`pod 'MapBox', :git => 'https://github.com/mapbox/mapbox-ios-sdk.git'` and 
`pod 'MapBox', :podspec => 'https://raw.github.com/mapbox/mapbox-ios-sdk/release/MapBox.podspec'`

![Screen Shot 2013-04-05 at 12 26 53 PM](https://f.cloud.github.com/assets/481100/345077/07f4b7a4-9e16-11e2-9811-8cb53e58b256.png)
![Screen Shot 2013-04-05 at 12 22 19 PM](https://f.cloud.github.com/assets/481100/345078/0ad64190-9e16-11e2-8c73-4309c17ff2d7.png)

**Comment by incanus** [comment]
Still trying to get to the bottom of this. I see the problem as well with 0.17, but not before.

**Comment by SteffenJ** [comment]
I get the same error as Roman with cocoapods 0.18.1. libProj4.a doesn't seem to get linked to the project. As a workaround I am able to build my project if I link manually with this static library.

**Comment by hermanlintvelt** [comment]
I'm also getting this issue with CocoaPods 0.18.1, got it working using manual link to libProj4.a as suggested by SteffenJ, but of course that is only interim.

**Comment by toddrbryan** [comment]
Same error here too - Pods 0.17.2.

**Comment by mkrus** [comment]
same here with pods 0.18.1

**Comment by activeintime** [comment]
Same

**Comment by activeintime** [comment]
Adding libProj4.a also fixed this for me

**Comment by incanus** [comment]
Bunch of upheaval in the CocoaPods spec format. I've just fixed this for 1.0.0+ in `develop` (e5dcde8d370b1f0e32a8359fb0858e7e8280614a), `release` (e5119cb13bebbd497ee0ab43fd1d9c1b8d42ffc0), and the master specs repo (https://github.com/CocoaPods/Specs/commit/d8bb6fd4bc8132ed00489b7635babfb0bbf89429), for 0.18+.
